www/offer_schemas/QmX6CZ7Wf8B79EX5x1PJSityvhtvvFKhkDBCDZK2cd6adF.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title": "Simple Ecommerce Schema",
+    "description": "Payload for buyer's metadata when the request to purchase an item.",
+    "type": "object",
+    "properties": {
+        "shippingAddress": {
+            "title": "Shipping Address",
+            "description": "Buyer's address to ship the item to.",
+            "type": "string"
+        },
+        "email": {
```

Comment:
Eventually we should break address up into its component fields. Right now we can add validation to the email field.

Reply:
Ok I added in email validation and breaking up address is probably out of scope for this PR